### DISCUSSION TITLE
Custom Domain Ruins CSS

### DISCUSSION THREAD

**Discussion by yabberish** [community/community#22998] [Other Feature Feedback, Questions, & Ideas]
<p>Hello!</p>
<p>So I´ve been getting a problem where whenever I put my custom domain to github pages, it ruins the CSS, when I remove the custom domain and make it normal (<a href="http://github.io" rel="nofollow noopener">github.io</a>) it goes back perfectly.</p>
<p>my custom domain is <a href="https://elflanded.wtf" rel="nofollow noopener">https://elflanded.wtf</a></p>
<p>the repo is <a href="https://github.com/Elflanded/landing" rel="nofollow noopener">https://github.com/Elflanded/landing</a></p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/elflanded">@elflanded</a>,</p>
<p>Thank you for being here! I think your page is loading okay for me, are you still having issues?</p>

**Comment by yabberish** [comment, marked as answer]
<p>No not anymore, I emailed support, and got the issue fixed!</p>

**Reply by wh33les** [reply]
What fixed the issue?  I am having the same problem.
